### ISSUE TITLE
Dev Tools UI: Bad readability of props of type function

### ISSUE THREAD

**Issue by Aarbel** [facebook/react#17781]
Dev Tools evolved a lot the last months.

Last version doesn't display well functions in props, it leaves an empty field so hard to see if the prop is really passed and if there's really and function passes in the props of the component.

For example  i dont' know if the prop `onChange` really exists as a prop of the component `TokenInput`.
Don't know also which props are undefined.

Could be also great to use the propTypes to show the types of the props.

![image](https://user-images.githubusercontent.com/25119847/71781247-b5e8fb00-2fcc-11ea-9a08-86f7b772a138.png)

![image](https://user-images.githubusercontent.com/25119847/71781235-8c2fd400-2fcc-11ea-8c3f-fb0afe3ab8fc.png)

**Comment by bvaughn** [comment]
Can you share the code (not a picture of code, but actual runnable code) that reproduces the empty function names like `onAdd`, `onChange`, `onRemove` in your screenshot? Chrome usually does a pretty good job of assigning functions names, and DevTools just displays the name that's assigned by a browser.

**Comment by Aarbel** [comment]
```JS
class LabelsInput extends React.Component {

    onCompletionClick = (completion) => {
        console.log('HELLO')
    }

    onChange = (event) => {
        console.log('HELLO')
    }

    onTokenRemove = (token) => {
        console.log('HELLO')
    }

    onTokenCreationAndAdd = (completion, color) => {
        console.log('HELLO')
    }

    onEditAll = () => {
        console.log('>>> WOHOO onEdit');
    }

    render() {
        const { query } = this.state;
        const { placeholder, labels } = this.props;

        const tokenName = 'Créer le tag';
        const onEditAllText = 'Éditer les tags';

        return ( 
            <TokenInput 
                text={query}
                tokens= {this.tokens}
                completions={this.completions}
                placeholder={placeholder}
                onChange={this.onChange}
                onAdd={this.onCompletionClick}
                onRemove={this.onTokenRemove}
                tokenName={tokenName}
                handlesColor={true}
                onTokenCreationAndAdd={this.onTokenCreationAndAdd}
                onEditAll={this.onEditAll.bind(this)}
                onEditAllText={onEditAllText}
            />
        );
    }
}
```

**Comment by bvaughn** [comment]
More succinct, runnable repro:
https://codesandbox.io/s/mystifying-chaplygin-mbmtp

**Comment by bvaughn** [comment]
Fixed via #17789

Will go out with the next release, probably sometime this week

## Before
![Screen Shot 2020-01-06 at 9 13 29 AM](https://user-images.githubusercontent.com/29597/71835187-c9947000-3065-11ea-98cc-6327881ed506.png)

## After
![Screen Shot 2020-01-06 at 9 02 43 AM](https://user-images.githubusercontent.com/29597/71835182-c4372580-3065-11ea-9c75-71b51c8ff2a8.png)

**Comment by Aarbel** [comment]
Wow very fast, thanks a lot @bvaughn
